cmake/kconfig.cmake

```diff
@@ -28,21 +28,23 @@ set(kconfig_target_list
 
 set(COMMAND_FOR_config     ${KCONFIG_CONF} --oldaskconfig ${KCONFIG_ROOT})
 set(COMMAND_FOR_gconfig    gconf                          ${KCONFIG_ROOT})
-set(COMMAND_FOR_menuconfig ${KCONFIG_MCONF}               ${KCONFIG_ROOT})
```

Comment:
IMHO dropping mconf should be done later, but if we do it in this PR, then we need to update the docs:

http://docs.zephyrproject.org/application/application.html#overriding-default-configuration

and

getting_started/installation_win.rst
216:      pacman -S git cmake make gcc dtc diffutils ncurses-devel python3 gperf

Reply:
If we drop it later then the docs will be inconsistent for a while, that's why I suggested taking the leap directly in this PR

Reply:
Yeah, that's why I didn't want to update all the documentation right away either, because the original plan was to get rid of it separately.

I could go either way though. It's up to you and @carlescufi. :P